js/exporter/jspdf/export.js

```diff
@@ -125,24 +123,23 @@ export const Export = {
         }
     },
 
-    setColumnWidths: function(autoTableOptions, columns, keepColumnWidths) {
-        const columnStyles = autoTableOptions.columnStyles;
-
-        for(let i = 0; i < columns.length; i++) {
-            columnStyles[i] = columnStyles[i] || {};
-
-            const columnWidth = columns[i].gridColumn.width;
-            if(keepColumnWidths && (typeof columnWidth === 'number') && isFinite(columnWidth)) {
-                columnStyles[i].cellWidth = this.convertPixelsToPoints(columnWidth);
-            } else {
-                columnStyles[i].cellWidth = 'auto';
-            }
+    getDefaultPdfColumnWidths(autoTableWidth, columnWidths) {
+        if(isNumeric(autoTableWidth)) {
+            const tableWidth = columnWidths.reduce((a, b) => { return a + b; });
```

Comment:
What about the situation when **columnWidths** value is **undefined**. The getColumnsWidths() function may return such value. I think that we should add the follow condition 

 if(!isDefined(columnWidths)) {
    return;
}

Reply:
'isDefined' check is added